collect_app/src/main/java/org/odk/collect/android/preferences/ServerPreferencesFragment.java

```diff
@@ -453,8 +456,27 @@ private void runSmsPhoneNumberValidation() {
 
                 showDialog(alertDialog, getActivity());
             } else {
-                continueOnBackPressed();
+                runGoogleAccountValidation();
             }
+        } else {
+            runGoogleAccountValidation();
+        }
+    }
+
+    private void runGoogleAccountValidation() {
+        String account = (String) GeneralSharedPreferences.getInstance().get(KEY_SELECTED_GOOGLE_ACCOUNT);
+        String protocol = (String) GeneralSharedPreferences.getInstance().get(KEY_PROTOCOL);
+
+        if (TextUtils.isEmpty(account) && protocol.equals(getString(R.string.protocol_google_sheets))) {
+
+            AlertDialog alertDialog = new AlertDialog.Builder(getActivity())
+                    .setIcon(android.R.drawable.ic_dialog_info)
```

Comment:
I think for new dialogs that get added we can do away with the icon -- it doesn't look so modern! We should probably start building a design guidelines document.

Reply:
Yeah true! That document could help. I used it for consistent reasons so when do that document we could have some more modern choices and then just create an issue where all of them could be changed.